backends/ceed-backend-list.h

```diff
@@ -19,6 +19,8 @@ CEED_BACKEND(CeedRegister_Cuda_Shared, 1, "/gpu/cuda/shared")
 CEED_BACKEND(CeedRegister_Hip, 1, "/gpu/hip/ref")
 CEED_BACKEND(CeedRegister_Hip_Gen, 1, "/gpu/hip/gen")
 CEED_BACKEND(CeedRegister_Hip_Shared, 1, "/gpu/hip/shared")
+CEED_BACKEND(CeedRegister_Sycl, 2, "/gpu/sycl/ref", "/cpu/sycl/ref")
+CEED_BACKEND(CeedRegister_Sycl_Shared, 2, "/gpu/sycl/shared", "/cpu/sycl/shared")
```

Comment:
ah, then we should update here to drop that for now if we don't have those as options yet

Reply:
Do we actually not have those? The dispatch code for `/cpu/sycl` stuff is still present and I recall we tested it early in this development process. I'd hope it still runs, and in fact, we might be able to test it on noether (using the `/opt/intel/oneapi` stack).

Reply:
They're not present when I build on Sunspot. I'm not sure if there's just a single line that got commented out/deleted to remove that functionality.

Reply:
Would I just need to add the cpu backends to the `SYCL_BACKENDS` list in the Makefile to get it?

Reply:
What do you mean by "not present"? Not reported in `make info` or fails saying there is no such backend when running with `-ceed /cpu/sycl`?

Reply:
Not reported by make info.

Reply:
Let's try dropping that last commit and see if it runs. Could also test on noether. Then we can add it to `SYCL_BACKENDS` so it shows up in `make info`.

Reply:
libCEED tests with `/cpu/sycl/*` backends report:

```
SKIP: sycl device type not available fluids-navierstokes /cpu/sycl/shared
```

Attempting to run a test outside of the testing framework (ie. `build/t570-operator /cpu/sycl/shared`) results in:
```
/home/jrwrigh/software/libCEED/backends/sycl/ceed-sycl-common.sycl.cpp:46 in CeedSyclInit(): No SYCL devices of the requested type are available
```

So it still might work, the backend is just blocking itself from trying.

*Side note:* Should that be `/cpu/self/sycl/*`? I see some cpu backends including `self` and others not. Can't quite see the pattern between the two.

Reply:
Currently the SYCL runtime compilation (even for qfunctions in sycl-ref) is hardwired for use with the Level Zero runtime, so the `/cpu/sycl/` backends should definitely be dropped from the Makefile and registration now. This can be relaxed in the future.